buildSrc/build.gradle.kts

```diff
@@ -27,6 +27,8 @@ dependencies {
   implementation(gradleApi())
   implementation(localGroovy())
 
+  implementation("io.opentelemetry.instrumentation.gradle:codegen:1.4.0-SNAPSHOT")
```

Comment:
Just in case it wasn't obvious, the dance we need to perform is 1) comment out this line 2) rename .gradle.kts.bak to .gradle.kts 3) ./gradlew :codegen:publishToMavenLocal 4) Undo 1+2